fixtures/reconciler/README.md

```diff
@@ -0,0 +1,17 @@
+# React Reconciler Test Fixture
+
+This folder exists for **React contributors** only.
+If you use React you don't need to worry about it.
+
+These fixtures are a smoke-screen verification that the built React Reconciler distributions do not
```

Comment:
That's not what it does anymore.

Reply:
No? I guess now it just tests that “it works”?

Reply:
Yea.